eng/versioning.targets

```diff
@@ -22,6 +22,22 @@
     </AssemblyMetadata>
   </ItemGroup>
 
+  <!-- Assembly metadata indicating that an assembly should be trimmed by ILLink (during a self-contained publish - this
+       is independent of whether we link during the runtime build). May be set by test projects.
+       The default is to include this metadata for netcoreapp assemblies but not OOB assemblies. -->
+  <PropertyGroup Condition="'$(IsTestProject)' != 'true' and '$(IsTestSupportProject)' != 'true' and ('$(IsNETCoreAppRef)' == 'true' or '$(IsNETCoreAppSrc)' == 'true')">
+    <SetIsTrimmable Condition="'$(SetIsTrimmable)' == ''">true</SetIsTrimmable>
+  </PropertyGroup>
+  <!-- IsTrimmable metadata is only supported for net6.0+. Don't include it when publishing for netfx or older TFMs. -->
+  <ItemGroup>
+    <AssemblyMetadata Include="IsTrimmable"
+                      Condition="'$(SetIsTrimmable)' == 'true' and
+                                '$(TargetFrameworkIdentifier)' == '.NETCoreApp' and
+                                $([MSBuild]::VersionGreaterThanOrEquals($(TargetFrameworkVersion), '6.0'))">
```

Comment:
This isn't going to work. We build assemblies for netstandard that we will want to make trimmable.

Reply:
N/A (check was removed)